buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -956,7 +956,7 @@ class BeamModulePlugin implements Plugin<Project> {
                 FileTree exposedClasses = project.zipTree(it).matching {
                   include "**/*.class"
                   // BEAM-5919: Exclude paths for Java 9 multi-release jars.
-                  exclude "META-INF/versions/*/module-info.class"
+                  exclude "**/module-info.class"
```

Comment:
This started ignoring "module-info.class".
Wiithout this change, we could have detected module-info.class sneaking into the vendor jar.